ui/client/components/right-panel/UserRightPanel.js

```diff
@@ -133,8 +133,8 @@ class UserRightPanel extends Component {
         {name: "migrate", visible: this.props.capabilities.deploy && !_.isEmpty(this.props.featuresSettings.remoteEnvironment), disabled: !conf.deployPossible, onClick: this.migrate, icon: InlinedSvgs.buttonMigrate},
         {name: "compare", onClick: this.compareVersions, icon: 'compare.svg', disabled: this.hasOneVersion()},
         {name: "import", visible: this.props.capabilities.write, disabled: false, onClick: this.importProcess, icon: InlinedSvgs.buttonImport, dropzone: true},
-        {name: "export", onClick: this.exportProcess, icon: InlinedSvgs.buttonExport},
-        {name: "exportPDF", disabled: !this.props.nothingToSave, onClick: this.exportProcessToPdf, icon: InlinedSvgs.buttonExport},
+        {name: "export", disabled: this.props.nothingToExport, onClick: this.exportProcess, icon: InlinedSvgs.buttonExport},
+        {name: "PDF", disabled: this.props.nothingToExport, onClick: this.exportProcessToPdf, icon: InlinedSvgs.pdf},
```

Comment:
Export PDF?

Reply:
Sorry, I forgot to include link to this issue: https://github.com/TouK/nussknacker/issues/171

Reply:
Okay, so leave it as PDF, but maybe in export we can put JSON?

Reply:
Oh, I have an idea: at title put full description like: Export to PDF file. And at rest of buttons you can add similar full description.

Reply:
Good idea, will apply that